apps/web/app/(app)/onboarding/OnboardingBulkUnsubscriber.tsx

```
@@ -0,0 +1,144 @@
+
+const useNewsletterStats = () => {
+  const now = useMemo(() => Date.now(), []);
+  const params: NewsletterStatsQuery = {
+    types: [],
+    filters: [],
+    orderBy: "unarchived",
+    limit: 5,
+    includeMissingUnsubscribe: false,
+    fromDate: now - ONE_MONTH_MS,
+  };
+  const urlParams = new URLSearchParams(params as any);
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)